### PR TITLE
Add undefined label prompt check

### PR DESCRIPTION
**Pull request by heyufan1995**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by bhashemian — COMMENTED** [review]
Looks good to me but please bump up the model version in metadata.json. Thanks

File: models/vista3d/scripts/evaluator.py [hunk 1]
```diff
@@ -166,6 +166,10 @@ def check_prompts_format(self, label_prompt, points, point_labels):
                     raise ValueError("Label prompt must be a list of single scalar, [1,2,3,4,...,].")
                 if not np.all([(x < 255).item() for x in label_prompt]):
                     raise ValueError("Current bundle only supports label prompt smaller than 255.")
+                if points is None:
+                    supported_list = list(set([i+1 for i in range(132)]) - set([16,18,129,130,131]))
+                    if not np.all([x in supported_list for x in label_prompt]):
+                        raise ValueError("Undefined label prompt detected. Provide point prompts for zero-shot.")
```
Comment (Nic-Ma) [inline models/vista3d/scripts/evaluator.py:172]:
```suggestion
                        raise ValueError("Undefined label prompt detected. Provide point prompts for zero-shot learning.")
```

**Review by Nic-Ma — APPROVED** [review]

File: models/vista3d/scripts/evaluator.py (hunk 1 above)
Comment (Nic-Ma) [reply models/vista3d/scripts/evaluator.py:172]: No need the change.

**Review by bhashemian — APPROVED** [review]

**Pull request merged**